<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="inputTextEdit">
        <property name="styleSheet">
            <string>background-color: #f0f0f0; color: #333; border: 1px solid #ccc;</string>
        </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="outputTextEdit">
        <property name="styleSheet">
            <string>background-color: #f0f0f0; color: #333; border: 1px solid #ccc;</string>
        </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="algorithmComboBox">
        <property name="styleSheet">
            <string>background-color: #fff; color: #333; border: 1px solid #ccc;</string>
        </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="encryptButton">
      <property name="text">
       <string>Encrypt</string>
      </property>
      <property name="styleSheet">
            <string>background-color: #4CAF50; color: white; border: none; padding: 10px 20px; text-align: center; text-decoration: none; display: inline-block; font-size: 16px; margin: 4px 2px; cursor: pointer; border-radius: 4px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="decryptButton">
      <property name="text">
       <string>Decrypt</string>
      </property>
      <property name="styleSheet">
            <string>background-color: #f44336; color: white; border: none; padding: 10px 20px; text-align: center; text-decoration: none; display: inline-block; font-size: 16px; margin: 4px 2px; cursor: pointer; border-radius: 4px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>

